content/blog/todo.md

```diff
@@ -28,37 +55,40 @@ Because if you don't have it you're going to end up firing them in a year and a
 
 Action is more important then whether you start with an idea or start with finding a problem to work on. Find what resonates with you and gets you excited.
 
-### LLM mental models
+### Digital nomad hacker house
+
+I want to learn more about this.
 
-Axe head, fine tuning, importance of prompts
+### I want to do more hackathons
 
-### Needs for tooling in the LLM space
+I want to do more of this and meet a lot more smart motivated people. Working with people smarter than you.
 
-So hard to work on prompt engineering
+### Product market fit
 
-### Future of LLMs
+If it isn't completely obvious to you that you have it then you don't have it. It should feel like you're being pulled out to ocean by a riptide.
 
-GPT-4 will be huge, will it be too expensive? Is building LLM's from scratch necessary or are we just not writing the right prompts. What will the next architecture be and will it involve some sort of semantic linking between inputs. I think LLM's are turning into commodities.
+I've never experienced true product market fit with anything I've built. The closest experience I've had is when one of my [posts](/blog/2022/productivity-porn/) went to #2 on Hacker News. In the span of a week more than 60k people read my post, aggregators around the internet were picking it up, and I received numerous emails from people with personal heartfelt messages. It felt like I was just along for the ride.
 
-### Digital nomad hacker house
+Ultimately when I'm building something the goal is product market fit. Remembering this feeling of being pulled vs. pushed will help me know when I've found product market fit and when I should keep looking.
 
-I want to learn more about this.
+## Life
 
-### I want to do more hackathons
+### The value of regular therapy or life coaching
 
-I want to do more of this and meet a lot more smart motivated people.
+I've seen therapists in acute contexts and it has been transformational. Correspondingly I've become a big advocate for therapy. However, I've always tied it to acute issues in my life. What if I integrated it more regularly into my life? This could be hugely beneficial since a programmer spends most of the day inside their own head. Speaking from personal experience I can say that dips in my mental health can wreak havoc on my productivity (not mentioning its effects on the rest of my life). It only makes sense that mental health should be regularly cared for like physical health.
 
 ### Threefold training mental health
 
 Talking with Ian
 
-### Respect the LLMs
-
-They are smarter than we give them credit for.
-
 ### The importance of ego and humbleness
 
 Stay low ego
 
+---
+
+_Special thanks to Andy, Ian, Greg, Brian, and Oliver for some of these insights._
+
 Andy Chou thought I was a sr. engineer
+
```

Comment:
I'm pretty sure this is also not the right place to be editing.  
I might need a personal tutorial....